src/KafkaFlow.OpenTelemetry/ExtensionMethods.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace KafkaFlow.Configuration
+{
+    using KafkaFlow.OpenTelemetry;
+
+    /// <summary>
+    /// Adds OpenTelemetry instrumentation
+    /// </summary>
+    public static class ExtensionMethods
```

Comment:
Is this a naming pattern for classes for extension methods? Seems too generic for me 😅